3_ProtectionProfile/PP config.adoc

```diff
@@ -21,15 +21,15 @@ The purpose of a PP-Configuration is to define a Target of Evaluation (TOE) that
 
 This PP-Configuration is identified as follows:
 
-* PP-Configuration for Protection Profile for Mobile Device Fundamentals and collaborative Protection Profile Module for Mobile biometric enrolment and verification - for unlocking the device - version 0.8, January 31, 2019
+* PP-Configuration for Protection Profile for Mobile Device Fundamentals and collaborative PP-Module for Mobile biometric enrolment and verification - for unlocking the device - version 0.8, May 01, 2019
 * As a shorthand reference, it can be identified as "BIO+MDF"
 
 === PP-Configuration Components Statement
 
 This PP-Configuration includes the following components:
 
 * Base-PP: Protection Profile for Mobile Device Fundamentals [MDFPP]
```

Comment:
The latest term agreed is "base PP"  (lower case and no hyphen)

Reply:
I will check all the other docs for this (I have been writing it like this for some time).

Reply:
This has been completed (for all docs)